manager/service.go

```diff
@@ -160,6 +164,22 @@ func (ms *managerService) Stop(ctx context.Context, computationID string) error
 	return nil
 }
 
+func (ms *managerService) FetchBackendInfo() ([]byte, error) {
+	cmd := exec.Command("sudo", "../../scripts/backend_info/backend_info", "--policy", "1966081")
```

Comment:
should be an env variable, and the default should be in the build dir as other bins. Also sudo is not required